nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -588,6 +589,63 @@ def modify_state_dict(self, conf, state_dict):
                 new_state_dict[new_key] = state_dict[key]
             state_dict = new_state_dict
 
+        # compatibility for inductor in inference
+        if not conf.get('inductor', False):
+            new_state_dict = {}
+            for key in state_dict.keys():
+                new_key = key.replace('._orig_mod', '', 1)
+                new_state_dict[new_key] = state_dict[key]
+            state_dict = new_state_dict
+
+        # Modify state key for Dreambooth inference
+        if (
+            conf.get('target')
```

Comment:
Exact str class path match is bad since if you refactor the path this is going to fail